src/components/DataView/example/DataViewKitchenSink.tsx

```diff
@@ -66,7 +66,7 @@ const processStringFilter = function ({ name, data, output }) {
 	if (data.comparison === "equals") {
 		output[name] = data.value;
 	} else if (data.comparison === "contains") {
-		output[name] = new RegExp(`.*${data.value}.*`, "i");
+		output[name] = data.value;
 	} else if (data.comparison === "not_contains") {
 		output[name] = new RegExp(`^((?!${data.value}).)*$`, "i")

```

Comment:
Not contains also needs to be just a string because then it messes with title with comparison.